<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расписание </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetTimeTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>681</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Понедельник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вторник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Среда</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Четверг</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пятница</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Суббота</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRun">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Расписание</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
